src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -60,6 +60,16 @@ import { isMacintosh, isWindows, isLinux, isWeb, isNative } from 'vs/base/common
 				],
 				'description': nls.localize({ comment: ['This is the description for a setting. Values surrounded by single quotes are not to be translated.'], key: 'tabSizing' }, "Controls the sizing of editor tabs.")
 			},
+			'workbench.editor.splitSizing': {
+				'type': 'string',
+				'enum': ['Distribute', 'Split'],
+				'default': 'Distribute',
+				'enumDescriptions': [
+					nls.localize('workbench.editor.splitSizing.Distribute', "Splits all the editors to equal parts."),
+					nls.localize('workbench.editor.splitSizing.Split', "Splits the active editor to equal parts.")
+				],
+				'description': nls.localize({ comment: ['This is the description for a setting. Values surrounded by single quotes are not to be translated.'], key: 'splitSizing' }, "Controls the sizing of editor tabs.")
```

Comment:
`Controls the sizing of editor tabs.` => `Controls the sizing of editors when splitting them.`